src/Faker/Provider/Image.php

```diff
@@ -22,6 +22,13 @@ class Image extends Base
         'fashion', 'people', 'nature', 'sports', 'technics', 'transport',
     ];
 
+    /**
+     * @var array
+     */
+    public static $imageFormats = [
```

Comment:
Can we add constants for this?

Reply:
Added constants, apparently the `placeholder` website wouldn't actually generate a `gif` format, so I removed it.